jans-cli-tui/cli_tui/jans_cli_tui.py

```
@@ -498,12 +494,14 @@ async def coroutine():
 
                     self.cli_object_ok = True
                     self.check_available_plugins()
+                    self.main_nav_selection_changed(self.nav_bar.navbar_entries[0][0])
 
                 asyncio.ensure_future(coroutine())
```

Ruff (0.15.12)

[warning] Store a reference to the return value of `asyncio.ensure_future`

(RUF006)